src/bundle/EventListener/RequestAttributesListener.php

```diff
@@ -60,6 +67,18 @@ public function addRequestAttributes(FilterViewBuilderParametersEvent $event)
             $parameterBag->remove('locationId');
             $parameterBag->set('location', $location);
         }
+
+        if ($this->fetchingTranslatedContent($request, $parameterBag)) {
+            /** @var Location $location */
+            $location = $parameterBag->get('location');
+            $content = $this->loadContent($location->contentInfo->id, $parameterBag->get('language'));
+            $parameterBag->set('content', $content);
+        }
+    }
+
+    private function fetchingTranslatedContent(Request $request, ParameterBag $parameterBag)
```

Comment:
I don't think this is the best method name. It's a good practice to name methods returning boolean value name with `is` or `has` prefix. In this case I believe `hasContentLanguage` would be more accurate.

You are missing return type hint here and there. Adding docblock wouldn't harm either.